src/libs/elektra/kdb.c

```diff
@@ -810,11 +817,16 @@ static int elektraCacheLoadSplit (KDB * handle, Split * split, KeySet * ks, KeyS
 	if (debugGlobalPositions)
 	{
 		keySetName (parentKey, keyName (initialParent));
-		elektraGlobalGet (handle, *cache, parentKey, PROCGETSTORAGE, INIT);
-		elektraGlobalGet (handle, *cache, parentKey, PROCGETSTORAGE, MAXONCE);
-		elektraGlobalGet (handle, *cache, parentKey, PROCGETSTORAGE, DEINIT);
+		elektraGlobalGet (handle, ks, parentKey, PREGETSTORAGE, INIT);
+		elektraGlobalGet (handle, ks, parentKey, PREGETSTORAGE, MAXONCE);
+		elektraGlobalGet (handle, ks, parentKey, PREGETSTORAGE, DEINIT);
```

Comment:
I haven't tested this PR yet, but it seems weird to me that you use `ks` here and `*cache` in line 825.